packages/graphql-dynamodb-transformer/src/DynamoDBModelTransformer.ts

```diff
@@ -380,8 +380,9 @@ export class DynamoDBModelTransformer extends Transformer {
       ctx.mapResourceToStack(typeName, resourceId);
 
       queryFields.push(makeConnectionField(listResolver.Properties.FieldName.toString(), def.name.value));
+      this.generateConditionInputs(ctx, def);
```

Comment:
Do we need to run this portion? The missing piece seems to be AttributeTypes could we add that generation inside `generateFilterInputs`?

Reply:
Good call, I'll make that change